vcpkg.json

```diff
@@ -0,0 +1,71 @@
+{
+    "name": "crow-examples",
+    "version": "0.2",
+    "dependencies": [
+        {
+            "name": "boost-array",
+            "version>=": "1.70.0" 
+        },
+        {
+            "name": "boost-algorithm",
+            "version>=": "1.70.0" 
+        },
+        {
+            "name": "boost-asio",
+            "version>=": "1.70.0" 
+        },
+        {
+            "name": "boost-date-time",
+            "version>=": "1.70.0" 
+        },
+        {
+            "name": "boost-functional",
+            "version>=": "1.70.0" 
+        },
+        {
+            "name": "boost-lexical-cast",
+            "version>=": "1.70.0" 
+        },
+        {
+            "name": "boost-optional",
+            "version>=": "1.70.0" 
+        },
+        {
+            "name": "openssl-windows" 
+        },
+        {
+            "name": "zlib"
+        }
+    ],
+    "overrides": [
+        {
+            "name": "boost-array",
+            "version": "1.70.0" 
+        },
+        {
+            "name": "boost-algorithm",
+            "version": "1.70.0" 
+        },
+        {
+            "name": "boost-asio",
+            "version-semver": "1.70.0-2" 
+        },
+        {
+            "name": "boost-date-time",
+            "version": "1.70.0" 
+        },
+        {
+            "name": "boost-functional",
+            "version": "1.70.0" 
+        },
+        {
+            "name": "boost-lexical-cast",
+            "version": "1.70.0" 
+        },
+        {
+            "name": "boost-optional",
+            "version": "1.70.0" 
+        }
+    ],
+    "builtin-baseline": "44d94c2edbd44f0c01d66c2ad95eb6982a9a61bc"
```

Comment:
I was wondering what `builtin-baseline` did, just for me to be able to maintain this file later on.

Reply:
That's the commit hash of the `vcpkg` repo.  See [here](https://vcpkg.readthedocs.io/en/latest/users/versioning/#builtin-baseline).

Reply:
understood, thanks